tests/performance/performance.cpp

```diff
@@ -0,0 +1,370 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#define CATCH_CONFIG_MAIN
+
+#include "performance_measure.h"
+
+#include "catch_wrapper.hpp"
+#include "ebpf.h"
+#include "ebpf_epoch.h"
+#include "ebpf_core.h"
+#include "ebpf_maps.h"
+#include "ebpf_object.h"
+#include "ebpf_program.h"
+#include "helpers.h"
+extern "C"
```

Comment:
I wonder if we should do a ubpf_wrapper.h that does this?

Reply:
It might make sense. Something to consider.